plugins/modules/remote_management/manageiq/manageiq_provider.py

```diff
@@ -571,7 +571,7 @@ def delete_nulls(h):
         a hash without nulls
     """
     if isinstance(h, list):
-        return map(delete_nulls, h)
+        return [delete_nulls(i) for i in h]
```

Comment:
```suggestion
        return [delete_nulls(i) for i in h if i is not None]
```
@kenchrcum @felixfontein  Don't we need to add this condition? (I looked at the "dict" condition in two lines below).

Reply:
The original module didn't have it, so if we add it here, we modify behavior and potentially break backwards compatibility.

Reply:
That was my first suggestion, too. But as @felixfontein stated out in his review None was only deleted if it was the value in a dict, not as an element in the array. So if we keep the previous behaviour and just want to fix this for Python 3 "if i is not None" is not needed.

Reply:
got it, thanks!